packages/salesforcedx-vscode-core/package.json

```diff
@@ -69,6 +69,7 @@
   },
   "activationEvents": [
     "workspaceContains:sfdx-project.json",
+    "workspaceContains:core-workspace.code-workspace",
```

Comment:
@nrkruk this is the new workspace file that will looking for. Let me know what's the final name we'll go with. I suggest we pick one that hints to be specific to us so we don't activate the extensions unnecessarily.

Reply:
Looking at this now, we don't actually need any changes here I don't think.